docs/source/index.rst

```diff
@@ -10,6 +9,43 @@ possible. We'll walk you through everything from setting up your development
 environment to understanding our code structure and contributing to the
 project.
 
+Project purpose
+###############
+This project helps bootstrapping an environment for RHOSO_, the Red Hat OpenStack Services
+on OpenShift.
+
+It's done in a 2-steps approach: first get the infrastructure, then deploy the product.
+
+This project is mostly CI, QE and Dev oriented, and isn't intended for anything linked to
+production.
+
+.. _RHOSO: https://www.redhat.com/en/blog/red-hat-openstack-services-openshift-next-generation-red-hat-openstack-platform
+
+Found a bug or need a feature?
+##############################
+
+The project is under constant development, bugs happen.
+
+If you have such a bad encounter, please fill an `issue in Jira`_.
+
+Chose **OSPRH**  project, add **cifmw** label, and set the Workstream to **CI Framework** and the Team to **OSP CI Framework**.
+
+Please provide the following information:
+
+- your environment (OS version, ``ansible --version``)
+- commit hash of the ci-framework repository you're using (``git log -1``)
+- task name raising the error
+- the error itself (please use the internal pastebin if it's too long)
+- the command you issued
+- your environment files (yes, *all* of them - you may want to ensure the **security** field of the issue is properly set to not leak internal data)
```

Comment:
I would move this to line 32, and I'll work internally to introduce a workflow that sets this automatically when these workstreams/labels are used

Reply:
sounds good, lemme update.